docs/mecsimcalc-library/plotdraw.md

```diff
@@ -3,167 +3,29 @@ sidebar_label: "Plot Draw"
 sidebar_position: 2
 ---
 
+# Plot Draw
+
 <div style={{ margin: '1em 0' }}>
   <label htmlFor="version-select" style={{ fontWeight: 'bold', marginRight: '10px' }}>Select Version:</label>
   <select id="version-select" onChange={(e) => window.location.href = e.target.value}>
-    <option value="">Latest Release (v0.1.9)</option>
-    <!-- Add more options as needed -->
+     <option value="">Latest Release (v0.2.0)</option>
+    <option value="https://github.com/MecSimCalc/MecSimCalc-utils/blob/v0.1.9/PLOTDRAW_README.md">v0.1.9</option>
   </select>
 </div>
 
-# Plot Draw
-
 This library is designed to provide a set of functions for drawing various types of plots, arrows, segments, and shapes using Matplotlib. These functions allow for customized plotting and annotation of graphical elements.
 
 ## General
 
-### draw_arrow
```

Comment:
What happened to all these functions?

Reply:
In the last meeting, we agreed there were too many similar/redundant functions, so we deleted a lot of them